net/keepalived/files/usr/bin/keepalived-rsync-inotify

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+[ $# -lt 3 ] && exit 1
+
+vrrp_instance=$1
+peer=$2
+rsync_dir=$3
+
+INOTIFY_ACTIONS="create delete modify move moved_to moved_from"
+ARGS=${INOTIFY_ACTIONS:+ -e ${INOTIFY_ACTIONS// / -e }}
+
+[ ! -d /tmp/keepalived ] && mkdir /tmp/keepalived
+fifo=/tmp/keepalived/inotifywait-$peer.fifo
+mkfifo "${fifo}" || exit 1
+
+/usr/bin/inotifywait -q -r --exclude '/\..+' -o $fifo -m $rsync_dir $ARGS 2>/dev/null &
+pid="$!"
+
+cleanup() {
+	[ -n "$pid" ] && kill $pid
+	rm -f $fifo
+}
+trap cleanup TERM INT
+
+while read -r dir action file; do
+	RSYNC_TARGET_DIR=$(echo "${dir}" | sed -e "s:${rsync_dir}::g")
+	ACTION=NOTIFY_SYNC TYPE=peer INSTANCE=$vrrp_instance NAME=$peer RSYNC_SOURCE="${dir}${file}" RSYNC_TARGET="${RSYNC_TARGET_DIR}${file}" /sbin/hotplug-call keepalived
```

Comment:
Can you break the line so that it is not so long?

I also think it wouldn't be bad if we could add a few more log messages / debug messages here.
I did that afterwards, to understand the code better, when what happens in the code.

I also think it would be good if we outsourced everything to a main function.
Then it looks the same as a C program?

You could also run shellcheck against this.

Reply:
Added log functions in `/lib/functions/keepalived/common.sh`. Default logging for info/err is added and a few more with debug-level logging.

By default log_debug would not log anything, if the `KEEPALIVED_DEBUG=1` variable is set in the script, debug logs would be logged in the logger.